### ISSUE TITLE
EVReflection valueForAny unkown type _StringCore

### ISSUE THREAD

**Issue by ClarkXP** [evermeer/EVReflection#28]
I did update to 2.12.1 version and when I call to toJsonString() function the app is crashing. Previously. in 2.11.1 version, the code worked smoothly.

The classes that I'm trying to convert are the following:

```
import EVReflection

class BoundRequest:EVObject{
    var action:String = "byBounds"
    var platform:String = "android"
    var types:[String] = []
    var bounds:Bound = Bound()
}

class Bound:EVObject {
    var ne:Coordinate = Coordinate()
    var sw:Coordinate = Coordinate()    
}

class Coordinate:EVObject{
    var lat:Double=0
    var lng:Double=0

    func getCoordinate()->CLLocationCoordinate2D{
        return CLLocationCoordinate2D(latitude: lat, longitude: lng)
    }

    func setLatLng(lat:Double!,lng:Double!){
        self.lat=lat
        self.lng=lng
    }

    func setLatLng(latLng:CLLocationCoordinate2D){
        self.lat = latLng.latitude
        self.lng = latLng.longitude
    }
}
```

**Comment by evermeer** [comment]
I could reproduce your issue. Strange that the unit tests did not pick this one up. I now first have to go to work. I will pick this up later today. Thanks for reporting this!

**Comment by evermeer** [comment]
There is something strange with the Mirror function. When adding the line:
var types:[String]? = ["a","b"]
The Mirror function will return an extra child with the label _core which is of type _StringCore
I have reported the issue to the Swift development team.
I will now ignore that extra child. I will push a new version after I do some more tests.

**Comment by evermeer** [comment]
A new version with tag 2.12.2 has been pushed to GitHub and cocoapods

**Comment by evermeer** [comment]
I just found the real cause of this issue. The Mirror was also called for the objects inside the array.  When doing a mirror on a string it will return an item _core of type _StringCore. 

In the previous version this was skipped because I checked for objects of type EVObject instead of NSObject. I added tests for a number of NS\* classes that will have NSObject as it's base class.

I pushed this as version 2.12.3 to GitHub and cocoapods
